_data/projects/pieces-example-ts.yml

```diff
@@ -0,0 +1,12 @@
+name: Pieces Example TS
+desc: This is an example project of getting started with Pieces OS ts package.
+site: https://github.com/pieces-app/example-ts
+tags:
+  - good first issue
+  - bug
+  - documentation
+  - community review
+  - design
```

Comment:
We don't allow spaces in tags, which is why there's parsing issues with this YAML file. 

Unless a tag is relevant to the project, I'd suggest removing these for now.